src/CCFieldValidator.js

```diff
@@ -2,13 +2,23 @@ import valid from "card-validator";
 import pick from "lodash.pick";
 import values from "lodash.values";
 import every from "lodash.every";
+import { CARDS_OVERRIDES } from "./cardsOverrides";
+
+const MAESTR0_BASE_LENGTH = 12;
+CARDS_OVERRIDES.forEach(card => valid.creditCardType.addCard(card));
 
 const toStatus = validation => {
   return validation.isValid ? "valid" :
          validation.isPotentiallyValid ? "incomplete" :
          "invalid";
 };
 
+const hasProperLength = cardNumber => cardNumber.split(" ").join("").length >= MAESTR0_BASE_LENGTH;
```

Comment:
Why do you have to split and then join again?

I think that aString.split("").join("") == aString

Reply:
I'm using those two combined to parse the cardNumber, remove whitespaces and put it all together in a single string.
I might as well use something like `cardNumber.replace(/\s/g,''")` to remove them, but I guess it's almost the same..